lib/geordi/commands/_setup_vnc.rb

```diff
@@ -2,47 +2,19 @@
 def _setup_vnc
   `clear`
 
-  Interaction.note 'This script will help you install a VNC server and a VNC viewer.'
+  Interaction.note 'This script will help you install a VNC viewer.'
   puts
   puts Util.strip_heredoc <<-TEXT
-    With those you will be able to use our cucumber script without being
-    disturbed by focus-stealing Selenium windows. Instead, they will open
-    inside a VNC session.
+    With that you will be able to use our docker vnc script to open
```

Comment:
`...to use our docker vnc script to open` -> `...to use our docker VNC script to open`